spec/controllers/autograders_controller_spec.rb

```diff
@@ -2,4 +2,84 @@
 
 RSpec.describe AutogradersController, type: :controller do
   render_views
+
+  describe "#create" do
+
+    user_id = get_admin
```

Comment:
try to use FactoryBot to create a course with an instructor, and assessment. See #1804 for more details (see the Controller Macros, and lines 81-87 in `spec/controllers/courses_controller_spec.rb`